### PR TITLE
byenow: Add version 0.2

### LINKED ISSUE #831: [Request] Add byenow
https://iobureau.com/byenow/

### PR DESCRIPTION
**Pull request by devhawk**
[byenow](https://iobureau.com/byenow/) is a command-line Windows utility for faster folder removal.

fixes #831

### CODE REVIEW

File: bucket/byenow.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.2",
+    "description": "Utility for folder removal",
+    "homepage": "https://iobureau.com/byenow/",
+    "license": "Freeware",
```
Comment (h404bi) [inline bucket/byenow.json]:
```suggestion
    "license": "BSD-2-Clause",
```
[BSD-2-Clause](https://github.com/apankrat/byenow/blob/master/LICENSE)

File: bucket/byenow.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.2",
+    "description": "Utility for folder removal",
```
Comment (h404bi) [inline bucket/byenow.json]:
```suggestion
    "description": "A command-line utility for faster folder removal.",
```

Comment (Ash258) [reply bucket/byenow.json]: @h404bi Dot is not needed. It is not sentence. `command line` is obsolete as it is in Main bucket

File: bucket/byenow.json (hunk 1 above)
Comment (Ash258) [reply bucket/byenow.json]: image

**Pull request merged**